src/v2/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -46,6 +46,30 @@ export interface Props {
 
 declare const window: any
 
+export class LegacyArtworkDllContainer extends React.Component {
```

Comment:
Can we add a comment above here describing whats going on?

Reply:
🧙 and ✨ , yes I will attempt to write a comment that provides value.

Reply:
👍 ty! If we can add a bit more detail about approach (using dll, etc) in the PR description (when out of WIP) that would be appreciated too.